README.md

```diff
@@ -1,4 +1,4 @@
-# DeviceTree for the Zephyr Project
+# nRF DeviceTree
 
 DeviceTree language support for the [Zephyr project](https://zephyrproject.org/) in VS Code.
```

Comment:
Perhaps this should be changed as well? Here and in kconfig

Reply:
Yes, I thought about it but came to conclusion that, in the end, the feature is in Zephyr.